NetworkOfPrivateClinics/NetworkOfPrivateClinics/Clinic.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace NetworkOfPrivateClinics
+{
+    public class Clinic
+    {
+        public static int ClinicID { get; private set; }
```

Comment:
Why do we use static field here. This way there are no unique clinics. Rework to not static